### PR TITLE
Solve the longest word puzzle

### PR DESCRIPTION
**Pull request by DGEs2018**
Could you please review my codes for this exercise ? Thank you !

### CODE REVIEW

File: challenges/longest-word.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/*
+Write a program that takes in a string parameter and returns the longest word.
+Assumptions 
+Input string is comprised of only letters and spaces.
+*/
+
 function longestWord(phrase) {
```
Comment (yhabib) [inline challenges/longest-word.js]: JSDoc 🙏 . Remember `/**` and then vsc will take care 💪

File: challenges/longest-word.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
 	// split the input sentence into separate words
 	const phraseSplitted = phrase.split(' ');
```
Comment (yhabib) [inline challenges/longest-word.js]: Yeeees 👌

File: challenges/longest-word.js (hunk 1 above)
Comment (DGEs2018) [reply challenges/longest-word.js]: I'll ensure to continuously use JSDoc from now on !

**Comment by yhabib** [comment]
Remember from now on to always use `console.assert` 💪

**Pull request merged**